symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py

```diff
@@ -0,0 +1,38 @@
+from sympy import Eq, solve
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless,)
+
+# Description
+## The atoms of the target material evaporate and move towards the substrate inside the magnetron. There is a distance at which the energy of the atomized
+## atom will decrease to the energy of the thermal motion of the atoms of the gas-discharge plasma. This distance is called the boundary of the thermalization zone.
```

Comment:
Atomized atom sounds weird.

Reply:
> Atomized atom sounds weird.

What's the best way? sprayed atom? pulverized atom?